plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -126,23 +124,43 @@ class GeneralServiceFormSection extends Component {
     }
 
     return (
-      <p>
-        {'Need to run a service with multiple containers? '}
-        <a className="clickable" onClick={this.handleOpenConvertToPodModal}>
-          Add another container
-        </a>.
-      </p>
+      <div className="pod pod-narrow pod-short">
```

Comment:
Not really, but it looks better when collapsed
![image](https://cloud.githubusercontent.com/assets/883486/22420604/7e645876-e6e3-11e6-8765-ae3fcacb0388.png)

It also helps differentiate it from the rest of the form when expanded

![image](https://cloud.githubusercontent.com/assets/883486/22420625/97e20f3c-e6e3-11e6-9c1a-d0801aac4257.png)